fsi-workflow/04_fluidSimulation/tasks.md

```diff
@@ -76,38 +76,40 @@ In order to run simulation, open a terminal from the `skeleton` folder, source O
 - running `simpleFoam` in parallel and logging the output in `log.solver`
 - reconstucting the latest timeStep
 
-## Monitoring
+By default, the script is running the case with 8 processes, using oversubscription. You can change the partitioning by changing the `system/decomposeParDict` and you then change the number of processes in `run_case.sh`.
+
+The simulation will probably take around 5-10 min to complete all 500 iterations. You can get a pretty much converged state earlier (e.g., around 300 iterations).
 
-(**TODO** check if available)
```

Comment:
Works for me, I will add pyFoam to the USB stick as well.

Reply:
can you do a screenshot to add to the task? Otherwise I'll do it tomorrow